DynamicTablesPkg/Include/ArmNameSpaceObjects.h

```diff
@@ -723,6 +727,59 @@ typedef struct CmArmEtInfo {
   ARM_ET_TYPE    EtType;
 } CM_ARM_ET_INFO;
 
+/** A structure that describes the Memory Initiator and Target number.
+
+    ID: EArmObjMemoryInitTargetInfo
+*/
+
+typedef struct CmArmMemoryInitTargetInfo {
+  UINT32 NumInitiator;
+  UINT32 NumTarget;
+} CM_ARM_MEMORY_INIT_TARGET_INFO;
+
+/** A structure that describes the Memory Proximity Domain Attribute.
+
+    ID: EArmObjMemoryProxDomainAttrInfo
+*/
+
+typedef struct CmArmMemoryProxDomainAttrInfo {
+  UINT16 Type;
+  UINT16 Flags;
+  UINT32 ProcessorProximityDomain;
+  UINT32 MemoryProximityDomain;

```

Comment:
Same comment, this should be a token referencing a `CM_ARCH_COMMON_MEMORY_AFFINITY_INFO` struct